app/src/main/java/net/bible/android/control/backup/BackupControl.kt

```diff
@@ -52,11 +53,7 @@ import org.crosswire.jsword.book.Book
 import org.crosswire.jsword.book.BookCategory
 import org.crosswire.jsword.book.Books
 import org.crosswire.jsword.book.sword.SwordBookMetaData
-import java.io.BufferedInputStream
-import java.io.File
-import java.io.FileInputStream
-import java.io.FileOutputStream
-import java.io.InputStream
+import java.io.*
```

Comment:
Just personal preference coming from my python background, but I don't really like importing *. 
I know IDE likes to do this automatically by default, but it's configurable in IDE settings.